pyproject.toml

```diff
@@ -0,0 +1,29 @@
+[build-system]
+requires = [
+    "setuptools>=42",
+    "wheel>=0.36.0",
+    "scikit-build==0.14.0",
+    "cmake>=3.24.2",
+    "ninja; platform_system!='Windows'"
+]
+
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "lagrange-open"
```

Comment:
Not `lagrange-adobe`?

Reply:
I don't have strong opinions, but `lagrange-open` seems to be more consistent with our naming convention.